core/server/master/src/main/java/alluxio/master/AlluxioSecondaryMaster.java

```diff
@@ -91,8 +89,7 @@ public static void main(String[] args) {
    * @throws IOException if any I/O errors occur
    */
   private void connectToUFS() throws IOException {
-    String ufsAddress = Configuration.get(PropertyKey.UNDERFS_ADDRESS);
-    UnderFileSystem ufs = UnderFileSystem.Factory.get(ufsAddress);
+    UnderFileSystem ufs = UnderFileSystem.Factory.getForRoot();
```

Comment:
@ifcharming @apc999 
Do you plan to fix this so that it works for multiple mount points?